src/runtime_src/core/tools/xbmgmt2/OO_ChangeBoot.cpp

```diff
@@ -0,0 +1,101 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright (C) 2022 Advanced Micro Devices, Inc. All rights reserved.
+
+// ------ I N C L U D E   F I L E S -------------------------------------------
+// Local - Include Files
+#include "OO_ChangeBoot.h"
+
+// XRT - Include Files
+#include "core/common/query_requests.h"
+#include "tools/common/XBUtilities.h"
+#include "tools/common/XBUtilitiesCore.h"
+namespace XBU = XBUtilities;
+
+// 3rd Party Library - Include Files
+#include <boost/algorithm/string.hpp>
+#include <boost/format.hpp>
+#include <boost/program_options.hpp>
+namespace po = boost::program_options;
+
+// System - Include Files
+#include <fstream>
+#include <iostream>
+// =============================================================================
+
+// ----- H E L P E R M E T H O D S ------------------------------------------
```

Comment:
This is C++ and not Java :-)
```suggestion
// ----- HELPER MEMBER FUNCTIONS ------------------------------------------
```
Do not use fancy spacing, it kills the spell-checking and the string search.